content/patterns/listbox/examples/css/listbox.css

```
@@ -44,15 +44,30 @@
 [role="option"] {
   position: relative;
   display: block;
-  padding: 0 1em 0 1.5em;
+  margin: 2px;
+  padding: 2px 1em 2px 1.5em;
   line-height: 1.8em;
+  cursor: pointer;
 }
 
-[role="option"].focused {
+[role="listbox"]:focus [role="option"].focused {
   background: #bde4ff;
 }
 
-[role="option"][aria-selected="true"]::before {
+[role="listbox"]:focus [role="option"].focused,
+[role="option"]:hover {
+  outline: 2px solid currentcolor;
+}
+
+.move-right-btn span.icon::after {
+  content: " →";
+}
+
+.move-left-btn span.icon::before {
+  content: "← ";
+}
+
+[role="option"][aria-selected="true"] span.icon::before {
```

```suggestion
[role="option"][aria-selected="true"] span.checkmark::before {
```

It looks like [this commit](https://github.com/w3c/aria-practices/pull/2622/commits/6983e65401858658f6db47545171dbd006f5f3a3) regressed the visibility of the checkmark icon.

---

fixed